l2geth/rollup/sync_service.go

```diff
@@ -599,17 +599,15 @@ func (s *SyncService) GasPriceOracleOwnerAddress() *common.Address {
 
 /// Update the execution context's timestamp and blocknumber
```

Comment:
The latest L1 blocknumber logic is now updated, it simply will see if the remote tip is greater than the known latest L1 block number and update it if so